pkg/gadgets/trace/tcpdrop/tracer/dropreasongen/Makefile

```diff
@@ -0,0 +1,10 @@
+KERNEL_VERSION ?= v6.2
+URL ?= https://raw.githubusercontent.com/torvalds/linux/$(KERNEL_VERSION)/include/net/dropreason.h
```

Comment:
It seems the `enum skb_drop_reason` used by the tracepoint is the one defined in include/linux/skbuff.h and not the one in include/net/dropreason.h. They are not defined in the same way, so the current code does not give the correct reason. I need to update that.